package.json

```diff
@@ -3,6 +3,7 @@
   "version": "2.0.1-beta.128",
   "description": "AXA Switzerland Web Patterns",
   "main": "stack/server.js",
+  "module": "lib",
```

Comment:
Guess this expect a single module, not a folder containing multiple modules 🤔 
https://github.com/rollup/rollup/wiki/pkg.module

Reply:
https://github.com/rollup/rollup/issues/1593